buildSrc/src/main/java/Dependencies.kt

```diff
@@ -227,6 +227,7 @@ object Deps {
 
     const val junit = "junit:junit:${Versions.junit}"
     const val mockk = "io.mockk:mockk:${Versions.mockk}"
+    const val mockk_android = "io.mockk:mockk-android:${Versions.mockk}"
```

Comment:
This library allows usages of mockk in Android tests! Super helpful in certain cases imo